src/script/hydra-eval-jobset

```diff
@@ -357,22 +358,32 @@ sub evalJobs {
     my @cmd;
 
     if (defined $flakeRef) {
-        @cmd = ("hydra-eval-jobs",
-                "--flake", $flakeRef,
-                "--gc-roots-dir", getGCRootsDir,
-                "--max-jobs", 1);
+        my $nix_expr =
+            "let " .
+            "flake = builtins.getFlake (toString \"$flakeRef\"); " .
```

Comment:
Afaik `builtins.getFlake` requires things to be impure.
There is a work around... https://github.com/nix-community/nixos-anywhere/blob/9ba099b2ead073e0801b863c880be03a981f2dd1/terraform/nix-build/nix-build.sh#L38